### ISSUE TITLE
YOLOv8 Conduct training，but labels.jpg not data

### ISSUE THREAD

**Issue by xiaolieat** [ultralytics/assets#93]
Hello, when I was training with YOLO, I used the following line of command for training, my dataset is a YOLO format dataset downloaded from kaggle, I don't know why after training, there is only the number of categories in the labels.jpg, and there is no data for the other three graphs

yolo train data=YOLO_format/data.yaml model=ultralytics/cfg/models/v8/yolov8n.yaml pretrained=yolov8n.pt epochs=100 batch=64 resume=True

Here's a picture from my labels.jpg
![Image](https://github.com/user-attachments/assets/95747f3b-b573-4d12-9f57-c18416ebdc59)

Here's my version of YOLO
![Image](https://github.com/user-attachments/assets/549be044-752c-445b-bb15-f0de9bcc3921)

Here's a configuration file for my dataset
![Image](https://github.com/user-attachments/assets/10f8168f-e818-4acc-9e4a-fa695b6e12a0)

**Comment by UltralyticsAssistant** [comment]
👋 Hello @xiaolieat, thank you for submitting a `ultralytics/assets` 🚀 Issue. To help us address your concern efficiently, please ensure you've provided the following information:

1. For bug reports:
   - A clear and concise description of the bug (which you’ve provided, great! 😊)
   - A minimum reproducible example (MRE) that demonstrates the issue. This could include the dataset or commands you're using. You can refer to our guide on creating an [MRE](https://docs.ultralytics.com/help/minimum-reproducible-example/) for more details.
   - Your environment details (OS, Python version, package versions, etc.)
   - Expected behavior vs. actual behavior
   - Any error messages or logs—if applicable

It seems your description mentions the issue with `labels.jpg` during training, as well as related configuration details. Thank you for attaching images and providing some context! However, if possible, could you also share:
   - The content of your `data.yaml` file you’re using for training
   - Any logs/output shown during training, if errors or warnings are present

2. For general questions or further clarifications:
   - Provide as much context as possible about what you're trying to achieve.
   - Include any research you've done on this so far.
   - Specify which parts of our [documentation](https://docs.ultralytics.com/) you’ve already referred to.

Additionally, please ensure that your dataset adheres to the YOLO format described in the documentation. If you haven't already, you might find our dataset preparation guide useful!

Lastly, if this issue relates to the functionality of the YOLOv8 model, remember that this repository is specifically dedicated to assets. 😄 For training and inference-related issues, you might want to refer to the [`ultralytics/ultralytics`](https://github.com/ultralytics/ultralytics) repository.

This is an automated response to guide you efficiently, but don’t worry—an Ultralytics engineer will assist you soon. 🚀 Thank you for helping us improve!

**Comment by xiaolieat** [comment]
Here's a picture of my training,He can train normally, but there is only one image in the top left corner of labels.jpg, and the other three images are empty.Here's what I left to interrupt
![Image](https://github.com/user-attachments/assets/7e78f327-5679-4f46-b62c-1b46d4260546)

![Image](https://github.com/user-attachments/assets/f01ced25-9c00-4316-af6d-9f5b168e7543)

![Image](https://github.com/user-attachments/assets/cd650bc6-cedc-47bb-abdb-fb01b372e8b0)

**Comment by pderrenger** [comment]
@xiaolieat thanks for including details. The empty plots typically indicate missing or invalid labels in your validation set. Let's verify:

1. Ensure your `data.yaml` correctly points to both `train` and `val` image directories with corresponding label files
2. Confirm labels exist for all validation images (no empty .txt files needed for zero-label images)
3. Try updating to the latest Ultralytics package: `pip install -U ultralytics`

For dataset verification, you can run:  
`yolo checks` or `yolo check data=YOLO_format/data.yaml`

If issues persist, please share your dataset structure (number of images/labels in train/val) and consider testing with the standard COCO8 dataset from our Object Detection Datasets Guide to isolate the issue.
